css-box-model/styles.css

```diff
@@ -0,0 +1,19 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+  border-radius: 6px;
```

Comment:
These should all be applied to `button.language` instead. We don't want every single item to get this styling, only our buttons.